contracts/staking/validatorShare/ValidatorShare.sol

```diff
@@ -180,29 +203,33 @@ contract ValidatorShare is IValidatorShare, ERC20NonTransferable, OwnableLockabl
         return _amount;
     }
 
-    function sellVoucher(uint256 _minClaimAmount) public {
-        uint256 shares = balanceOf(msg.sender);
-        require(shares > 0, "Zero balance");
+    function sellVoucher(uint256 claimAmount) public {
```

Comment:
Shouldn't we put `minimumShares` with `claimAmount`?

Reply:
Depends how we want it to be. I can do it in any way.

Reply:
Actually, minimum shares doesn't make sense, because shares doesn't get slashed at all. Only amount of tokens can be lowered.

Reply:
but claimAmount to shares can be changed. So `shares` changes up or down just before calling `sellVoucher`?